examples/disaggregated_prefill_v1/load_balance_proxy_server_example.py

```diff
@@ -0,0 +1,435 @@
+# Adapted from https://github.com/vllm-project/vllm/tests/v1/kv_connector/nixl_integration/toy_proxy_server.py
+
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
it's good to add some usage guide here to let users know how to run this exmaple in quick